docker-compose.yml

```
@@ -9,4 +9,4 @@ services:
     environment:
       POSTGRES_DB: warn_me
       POSTGRES_USER: postgres
-      POSTGRES_PASSWORD: password
\ No newline at end of file
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
```

Why do you need to change password in docker-compose?  This file is only for starting application locally

```suggestion
      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
```